composer/2022_airflow_summit/data_analytics_dag.py

```diff
@@ -33,14 +33,14 @@
 
 # Dataproc configs
 BUCKET_NAME = "{{var.value.gcs_bucket}}"
-PYSPARK_JAR = "gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar"
 PROCESSING_PYTHON_FILE = f"gs://{BUCKET_NAME}/data_analytics_process.py"
 
-
 BATCH_ID = "data-processing-{{ ts_nodash | lower}}"  # Dataproc serverless only allows lowercase characters
 BATCH_CONFIG = {
+    "runtime_config": {
+        "version": "1.1"  # Will need to be changed to 2.0+ when that's GA
```

Comment:
Technically 2.0 is already GA but doesn't yet support the Spark-BigQuery connector. 1.1 is LTS until August 2025 so I think you can safely remove this comment or mention it should be revisited then.